admin/server/app/createStaticRouter.js

```diff
@@ -84,13 +87,12 @@ module.exports = function createStaticRouter (keystone) {
 
 	/* Configure router */
 	router.use('/styles', less(path.resolve(__dirname + '/../../public/styles'), lessOptions));
-	router.use('/styles/fonts', express.static(path.resolve(__dirname + '/../../public/js/lib/tinymce/skins/keystone/fonts')));
+	router.use('/styles/fonts', express.static(`${keystoneTinymcePath}/skin/fonts`));
 	router.get('/js/fields.js', bundles.fields.serve);
 	router.get('/js/signin.js', bundles.signin.serve);
 	router.get('/js/admin.js', bundles.admin.serve);
-	var keystone_tinymce = path.dirname(require.resolve('keystone-tinymce'));
```

Comment:
Realised I was using the same path above, so I removed and renamed this one 😬 

Let me know if it should be changed back